src/calculateRentalCost.js

```diff
@@ -4,7 +4,27 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let carRent = PRICE_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    carRent = carRent - LONG_TERM_DISCOUNT;
+
+    return carRent;
```

Comment:
The current logic applies the long-term discount and returns immediately, which prevents the short-term discount from being considered if the rental days are greater than or equal to the long-term threshold. Consider restructuring the logic to ensure that the correct discount is applied based on the number of rental days.